commons/libs/constants/GenericCodes.ts

```diff
@@ -1,12 +1,13 @@
 // TODO: add  error categories and codes as necessary with further development. remove this comment when the all the codes needed for application are mostly covered
-export const GenericCodes = {
-  unauthorized: 401,
-  notFound: 404
+export const GENERIC_CODES = {
+  UNAUTHORIZED: 401,
+  NOT_FOUND: 404,
+  ERROR: 500
 }
 
-export const HttpCodes = {
-  ...GenericCodes,
-  ok: 200,
-  created: 201,
-  noContent: 204
+export const HTTP_CODES = {
```

Comment:
Can't we use this?
https://nodejs.org/api/http.html#http_http_status_codes

Reply:
As far as I understand, `http.STATUS_CODES` returns a message when we pass a code to it. We are mapping the message to the code, so we can get the corresponding code by writing something like `HTTP_CODES.OK`.